pkg/agent/manager/cache/cache.go

```diff
@@ -557,13 +593,29 @@ func (c *Cache) notifyAll() {
 }
 
 func (c *Cache) notifyBySelectors(set selectorSet) {
+	c.log.WithFields(logrus.Fields{
```

Comment:
I'm on the fence about the usefulness v.s. noise on this new log line :) Did you intend to keep this in? If we do decide to keep it, we should probably change the debug message. Maybe something like "Notifying workloads by selectors".

Reply:
This is for debugging, not intended to be kept.